jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -3827,7 +3827,51 @@ static String valueOfCodePoint(int codePoint) {
 		}
 		return string;
 	}
-	/*[ENDIF] Java11*/
+
+
```

Comment:
Unnecessary double new line can be removed (similarly at the end of the function).